examples/Python/PET/acquisition_model.py

```diff
@@ -151,6 +151,16 @@ def main():
     back_projected_image_as_array = back_projected_image.as_array()
     show_2D_array('Backprojection', back_projected_image_as_array[z,:,:])
 
+    acq_model.direct(image, 0, 4, simulated_data)
```

Comment:
Please add a comment say what `direct` does. Non-obvious terminology for most of us.

Reply:
Also for me the terminology is not obvious. Especially with the background term being sometimes added and sometimes not. Perhaps this needs to be changed so that at least for some people this is intuitive?